module/entities/helper.js

```diff
@@ -1187,8 +1201,119 @@ module.exports = class EntitiesHelper {
         }
     }
 
+     /**
+   * Push entities data to elastic search
+   * @method
+   * @name pushEntitiesToElasticSearch
+   * @name entityId - created or modified entity data.
```

Comment:
@deepakr123 - Description doesn't match, also change it to entities

Reply:
resolved